citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/AuthorityServiceTest.java

```diff
@@ -196,4 +198,37 @@ public void test_findByUri_noResults() {
         List<IAuthorityEntry> actual = managerToTest.findByUri(user, uri);
         Assert.assertTrue(actual.isEmpty());
     }
+    
+    @Test
+    public void test_create() {
+        String username = "user";
+        OffsetDateTime createdOn = OffsetDateTime.now();
+        AuthorityEntry entry = new AuthorityEntry();
+        IUser user = new User();
+        user.setUsername(username);
+        Mockito.when(entryRepository.save(entry)).thenReturn(entry);
+        try {
+            Thread.sleep(100);
+        } catch (InterruptedException e) {}
+        IAuthorityEntry actualEntry = managerToTest.create(entry, user);
+        Assert.assertEquals(username, actualEntry.getUsername());       
+        Assert.assertTrue(createdOn.isBefore(actualEntry.getCreatedOn()));
+        Mockito.verify(entryRepository).save(entry);
+    }
+    
+    @Test
+    public void test_save() {
+        String id = "id";
+        String name = "name";
+        String uri = "uri";
+        AuthorityEntry entry = new AuthorityEntry();
+        entry.setId(id);
+        entry.setName(name);
+        entry.setUri(uri);
+        Mockito.when(entryRepository.save(entry)).thenReturn(entry);
+        
+        IAuthorityEntry actualEntry = managerToTest.save(entry);
+        Mockito.verify(entryRepository).save(entry);
+        Assert.assertEquals(entry, actualEntry);        
```

Comment:
This assumes that the method returns the object that the repository returns, which is not necessarily required (if the manager would create a new object and just copy all the values that would still be valid and the test case shouldn't fail). It would be cleaner to make sure the values are correct and not object equality.